src/App.js

```diff
@@ -51,9 +51,23 @@ function App() {
             </section>
           </Route>
 
+          <Route path="/Hogwarts">
+            <section className = "Potter">
+              <h1 id = "HP">hogwartsInformation.creature</h1>
+              <img
+                className = "Harry" 
+                src = {Sorting}
```

Comment:
Al intentar ver la card, aparece un error: `Sorting is not defined` porque no hay nada en el código con ese nombre. Para mostrar la imagen tendrías que haber agregado un `import Sorting from 'images/sortinghat.jpg;'` al inicio del archivo